src/main/java/sg/ncl/exceptions/MasterSubscriptionAgreementDownloadException.java

```diff
@@ -4,9 +4,17 @@
 /**
  * @author cassie
  */
+//public class MasterSubscriptionAgreementDownloadException extends Exception {
+//
+//    public MasterSubscriptionAgreementDownloadException(String message) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)